google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -249,6 +249,12 @@ def bucket bucket_name
         #   does not exist. For more information, see [How to Host a Static
         #   Website
         #   ](https://cloud.google.com/storage/docs/website-configuration#step4).
+        # @param [Boolean] requester_pays Indicates that a client accessing the
```

Comment:
It's acceptable to me. We can always add it later if there is sufficient demand.